go/vm/utils.go

```diff
@@ -66,3 +68,15 @@ func GetStorageStatus(original, current, new Word) StorageStatus {
 	// Default
 	return StorageAssigned
 }
+
+func Uint256ToValue(value *uint256.Int) (result Value) {
```

Comment:
As it is part of the `vm` package, these functions are part of the public interface of Tosca. Thus, they
- should be documented
- should have unit tests
Even if seemingly simple, byte-order issues may easily happen. Also, tests document their intended use.